package.json

```diff
@@ -76,6 +76,7 @@
     "stacktrace-gps": "^3.0.4",
     "style-loader": "^4.0.0",
     "text-field-edit": "^4.0.0",
+    "textarea-caret": "^3.1.0",
```

Comment:
This is missing a `PROVISION_VERSION` bump in `version.py`.

Reply:
Added a PR for updating: #30843.